java/spacewalk-java.spec

```
@@ -186,7 +186,7 @@ Requires:       glassfish-activation-api
 Requires:       glassfish-jaxb-api
 Requires:       glassfish-jaxb-runtime
 Requires:       glassfish-jaxb-txw2
-Requires:       google-gson >= 2.2.4
+Requires:       google-gson < 2.10.0
```

Like for the Buildrequires,  could this also work for you?
```suggestion
Requires:       (google-gson >= 2.2.4 with google-gson < 2.10.0)
```